brainstorm/src/index.tsx

```diff
@@ -41,7 +42,7 @@ export async function start() {
         ({ container, services } = await client.getContainer(containerId, containerSchema));
     }
 
-    if (!container.connected) {
+    if (container.connectionState !== ConnectionState.Connected) {
```

Comment:
Shouldn't this be expose thru fluid-framework as well? @scottn12 I assume that these examples shouldn't import package individually?

Reply:
This was a miss. We don't want azure developers pulling in other packages. @scottn12 I created [Bug 716](https://dev.azure.com/fluidframework/internal/_workitems/edit/716): Re-export ConnectionState from fluid-framework. Let's get this into the next minor release.

In the mean time...

```suggestion
    // TODO: Export ConnectionState type https://github.com/microsoft/FluidFramework/issues/10681
    if (container.connectionState !== 2 /*Connected*/) {
```

Reply:
@skylerjokiel can we add a policy check to make sure we don't accidentally import `inner packages` in these examples?

Reply:
AB#718